pages/resultset.py

```diff
@@ -17,21 +17,27 @@ class ResultsetPage(Base):
     _logviewer_button_locator = (By.ID, 'logviewer-btn')
     _resultset_locator = (By.CSS_SELECTOR, 'div.row.result-set')
     _result_status_locator = (By.ID, 'job-details-panel')
+    _unclassified_failure_count_locator = (By.CSS_SELECTOR, '.btn-unclassified-failures')
 
     @property
     def job_result_status(self):
         return self.selenium.find_element(*self._job_result_status_locator).text
 
+    @property
+    def verify_unclassified_failure_exists(self):
+        return self.selenium.find_element(*self.unclassified_failure_count_locator).text
+
     def go_to_page(self):
         self.open('')
 
     def open_next_unclassified_failure(self):
-        WebDriverWait(self.selenium, self.timeout).until(lambda s: self.selenium.find_element(*self._resultset_locator).is_displayed())
+        WebDriverWait(self.selenium, self.timeout).until(lambda s: self.selenium.find_element(*self._unclassified_failure_count_locator).is_displayed())
         self.selenium.find_element(*self._resultset_locator).send_keys("n")
 
     def open_logviewer(self):
         WebDriverWait(self.selenium, self.timeout).until(lambda s: self.selenium.find_element(*self._result_status_locator).is_displayed())
-        self.selenium.find_element(*self._resultset_locator).send_keys("l")
+        self.selenium.find_element(*self._logviewer_button_locator).click()
```

Comment:
If we want to test the keyboard shortcuts then first waiting for the button to be displayed might work. If we don't care about shortcuts vs buttons, then this change is fine with me.